pkg/systemd/index.html

```diff
@@ -446,10 +446,10 @@ <h4 class="modal-title"></h4>
                     <table>
                         <tr>
                             <td>
-                                <label translatable="yes">Delay</label>
+                                <label for="shutdown-delay-select" translatable="yes">Delay</label>
```

Comment:
This is also a bit premature -- there is a button in the custom dropdown below, but I don't think it would do a good job of being the target here. This first needs to replace our custom select below with a proper `<select>`.

Reply:
ack, I'll drop this one for now. I plan to migrate all custom dropdowns into native select soon-ish anyway.